Release process fails - Homebrew tap 403s

When cutting an Aria release for November, the GitHub automation errored out with:

```
Run gh workflow run main.yml -f version=v0.9.20251118 -R arialang/homebrew-aria
  gh workflow run main.yml -f version=v0.9.20251118 -R arialang/homebrew-aria
  shell: /usr/bin/bash -e {0}
  env:
    BIN_TARGETS: aria
    DYLIB_CRATES: aria_file aria_http aria_path aria_platform aria_regex aria_timezone
    EXTRA_FILES: CHANGELOG.md
    CARGO_TERM_COLOR: always
    CARGO_HOME: /home/runner/.cargo
    CARGO_INCREMENTAL: 0
    VERSION: v0.9.20251118
    GH_TOKEN: ***
could not create workflow dispatch event: HTTP 403: Resource not accessible by integration (https://api.github.com/repos/arialang/homebrew-aria/actions/workflows/203978280/dispatches)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -17,6 +17,11 @@ jobs:
       EXTRA_FILES: "CHANGELOG.md"
       CARGO_TERM_COLOR: always
     steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.ariahomebrewreleaseapp }}
+          private-key: ${{ secrets.ariahomebrewreleaseapp }}
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - name: Get version
@@ -34,6 +39,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Update homebrew tap
+        continue-on-error: true
         run: gh workflow run main.yml -f version=${{ env.VERSION }} -R arialang/homebrew-aria
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ steps.app-token.outputs.token }}

